contracts/root/Registry.sol

```diff
@@ -0,0 +1,31 @@
+pragma solidity ^0.5.5;
+
+import { Ownable } from "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+contract Registry is Ownable {
+  mapping(bytes32 => address) contractMap;
+
+  // @todo hardcode constants
+  bytes32 constant private WETH_TOKEN = keccak256('wethToken');
+  bytes32 constant private DEPOSIT_MANAGER = keccak256('depositManager');
+  bytes32 constant private STAKE_MANAGER = keccak256('stakeManager');
+
+  function updateContractMap(bytes32 _key, address _address)
+    public
+    onlyOwner
+  {
+    contractMap[_key] = _address;
+  }
+
+  function getWethTokenAddress() public view returns(address) {
```

Comment:
Wouldn't it better to have generic Registry?

Reply:
Probably, but then we'll need to embed the WETH_TOKEN's key everywhere we want to get it from registry.